here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -378,19 +385,14 @@ paths:
         '513':
           $ref: '#/components/responses/ErrorResponse513'
   '/hub/spaces/{spaceId}':
+    parameters:
```

Comment:
This is good idea to declare in parameters section and just referred from here. Let's follow the same for handlerId and storageId.